src/social_auth/south_migrations/0005_auto__chg_field_usersocialauth_extra_data.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+
+        # Changing field 'UserSocialAuth.extra_data'
+        db.alter_column(u'social_auth_usersocialauth', 'extra_data', self.gf('sentry.db.models.fields.encrypted.EncryptedJsonField')())
```

Comment:
🤔 Can you get the SQL and see what this is actually doing? I didn't realize this was from within social auth, so it's not using our code.

In theory, there shouldn't actually be any ALTER touching the database or we'll have a problem here.

Reply:
Yea, this won't work. I didn't know if South had some sort of data migration stuff built in, but it doesn't. This would have to be a bigger, staged, migration probably. Closing for now.